test/e2e/resources/components/BaseGuideView.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we should use dummy components for the testing and not this, it may not be meaningful for the open source community

Reply:
I agree with this